{"date":"Tue May 28 2019 18:09:55 GMT-0700 (PDT)","process":{"pid":47969,"uid":501,"gid":20,"cwd":"/Users/ariozone/code/vivideo-api","execPath":"/Users/ariozone/.nvm/versions/node/v10.0.0/bin/node","version":"v10.0.0","argv":["/Users/ariozone/.nvm/versions/node/v10.0.0/bin/node","/Users/ariozone/code/vivideo-api/index.js"],"memoryUsage":{"rss":66711552,"heapTotal":61714432,"heapUsed":27043128,"external":36104744}},"os":{"loadavg":[2.21728515625,2.36279296875,2.353515625],"uptime":688975},"trace":[{"column":11,"file":"/Users/ariozone/code/vivideo-api/node_modules/mongodb-core/lib/topologies/server.js","function":null,"line":564,"method":null,"native":false},{"column":13,"file":"events.js","function":"Pool.emit","line":182,"method":"emit","native":false},{"column":12,"file":"/Users/ariozone/code/vivideo-api/node_modules/mongodb-core/lib/connection/pool.js","function":null,"line":317,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":275,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Connection.emit","line":182,"method":"emit","native":false},{"column":50,"file":"/Users/ariozone/code/vivideo-api/node_modules/mongodb-core/lib/connection/connection.js","function":null,"line":246,"method":null,"native":false},{"column":13,"file":"events.js","function":"Object.onceWrapper","line":275,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"Socket.emit","line":182,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":92,"method":null,"native":false},{"column":3,"file":"internal/streams/destroy.js","function":"emitErrorAndCloseNT","line":59,"method":null,"native":false},{"column":19,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":174,"method":"_tickCallback","native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","    at Pool.<anonymous> (/Users/ariozone/code/vivideo-api/node_modules/mongodb-core/lib/topologies/server.js:564:11)","    at Pool.emit (events.js:182:13)","    at Connection.<anonymous> (/Users/ariozone/code/vivideo-api/node_modules/mongodb-core/lib/connection/pool.js:317:12)","    at Object.onceWrapper (events.js:275:13)","    at Connection.emit (events.js:182:13)","    at Socket.<anonymous> (/Users/ariozone/code/vivideo-api/node_modules/mongodb-core/lib/connection/connection.js:246:50)","    at Object.onceWrapper (events.js:275:13)","    at Socket.emit (events.js:182:13)","    at emitErrorNT (internal/streams/destroy.js:92:8)","    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)","    at process._tickCallback (internal/process/next_tick.js:174:19)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","timestamp":"2019-05-29T01:09:55.794Z"}
